pegasus/sites.v3/code.org/public/css/design-system-pegasus.scss

```diff
@@ -35,6 +35,12 @@ section.bg-primary-light {
   }
 }
 
+// Hide random whitespace at the bottom
+//  of the page from legacy code
+.container_responsive_full_width > .desktop-feature {
+  display: none;
+}
```

Comment:
Do we still need the code causing the whtiespace or can it be removed?

Reply:
It's used on legacy pages to add space above the footer — this should be done using margin on the `footer` element itself instead of inserting line breaks, but that's a sitewide change I haven't tackled yet 😁